backend/models.py

```
@@ -363,7 +362,9 @@ class TestCaseOutput(db.Model):
     __tablename__ = "test_case_outputs"
 
     id = db.Column(db.Integer, primary_key=True)
-    test_case_id = db.Column(db.Integer, db.ForeignKey("test_cases.id"), nullable=False, index=True)
+    test_case_id = db.Column(
+        db.Integer, db.ForeignKey("test_cases.id"), nullable=False, index=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)